### DISCUSSION TITLE
Port forwarding on Home Assistant OS

### DISCUSSION THREAD

**Discussion by Scholdan** [koush/scrypted#993] [Q&A]
Hello

I think there is missing the option to set which port to forward to the docker container

![image](https://github.com/koush/scrypted/assets/23090427/389c931d-5c51-49f1-a62e-9b65dfbeb928)

Example here from Lets Encrypt:

![image](https://github.com/koush/scrypted/assets/23090427/55cd43ce-3c03-4fb3-aaf1-384a877c54b5)

**Comment by koush** [comment]
Not sure what you're doing. None of the port forwarding stuff is necessary if you use the custom component.

https://docs.scrypted.app/home-assistant.html

**Comment by Scholdan** [comment]
It´s for your cloud plugin and not home assistant plugin!
